### PR TITLE
Update office-365-info-protection-for-gdpr-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed the Broken link in the document for Monitor for leaks of Personal Data

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3301

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/office-365-info-protection-for-gdpr-overview.md [hunk 1]
```diff
@@ -97,4 +97,4 @@ For more information about applying protection, see [Apply protection to persona
 
 ### Step 5 — Monitor for leaks of personal data
 
-Office 365 data loss prevention reports provide the greatest level of detail for monitoring sensitive data. You can setup automated alerts and investigate breaches by using the audit log. Cloud App Security extends the ability to find and monitor sensitive data to other SaaS providers. For more information on these tools, see [Monitor for breaches of personal data](/security/office-365-security/monitor-for-leaks-of-personal-data.md).
+Office 365 data loss prevention reports provide the greatest level of detail for monitoring sensitive data. You can setup automated alerts and investigate breaches by using the audit log. Cloud App Security extends the ability to find and monitor sensitive data to other SaaS providers. For more information on these tools, see [Monitor for breaches of personal data](https://docs.microsoft.com/microsoft-365/security/office-365-security/monitor-for-leaks-of-personal-data?view=o365-worldwide).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/office-365-info-protection-for-gdpr-overview.md]:
```suggestion
Office 365 data loss prevention reports provide the greatest level of detail for monitoring sensitive data. You can set up automated alerts and investigate breaches by using the audit log. Cloud App Security extends the ability to find and monitor sensitive data to other SaaS providers. For more information on these tools, see [Monitor for leaks of personal data](https://docs.microsoft.com/microsoft-365/security/office-365-security/monitor-for-leaks-of-personal-data?view=o365-worldwide).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@BrendaCarter Please merge this PR. Thanks

**Pull request merged**